boilerplate/openshift/golang-osd-operator/csv-generate/csv-generate.sh

```diff
@@ -93,6 +93,18 @@ else
     rm -rf "$SAAS_OPERATOR_DIR"
     git clone --branch "$operator_channel" ${GIT_PATH} "$SAAS_OPERATOR_DIR"
 
+    # First exit appropriately if the clone could not succeed
+    if [[ $? != 0 ]]; then
```

Comment:
Since this script is running under `set -e` (L3), I would expect it to exit right away if the `git clone` on L94 fails.

Reply:
Thanks a lot for the review @2uasimojo ! And thanks for the catch - I've removed that superfluous check now.